### PR TITLE
Added Phantom Solder PCB and KBD8X HS PCB Variants

### PR DESCRIPTION
**Pull request by mrnoisytiger**
Added Phantom Solder PCB variant and upcoming KBD8X HS PCB variants. 

For Phantom Solder, did not include multiple layout options as VIA is the expected use case.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lucid/phantom_solder/rules.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+KEYBOARD_LOCK_ENABLE = yes
+
+LAYOUTS = 65_all
```
Comment (drashna) [inline keyboards/lucid/phantom_solder/rules.mk:28]: This doesn't appear to be used. 
```suggestion
```

**Review by drashna — COMMENTED** [review]
Could you switch the `LAYOUT_ALL` to `LAYOUT_all`.  Anything after the `LAYOUT` part should be lower case.

**Comment by mrnoisytiger** [comment]
@drashna Changes made! Did not realize that convention, thanks for clarifying.

File: keyboards/lucid/phantom_solder/readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Phantom Solder PCB by Lucid
+
+The following is the QMK Firmware for the Phantom *Solder* PCB for [LucidKB](http://www.lucidkb.cm).
+---
+
```
Comment (zvecr) [inline keyboards/lucid/phantom_solder/readme.md:5]:
```suggestion

```

File: keyboards/lucid/phantom_solder/rules.mk [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+KEYBOARD_LOCK_ENABLE = yes
+
+LAYOUTS = all
```
Comment (zvecr) [inline keyboards/lucid/phantom_solder/rules.mk:28]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/lucid/kbd8x_hs/rules.mk [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+KEYBOARD_LOCK_ENABLE = yes
+
+LAYOUTS = tkl_ansi tkl_ansi_7u tkl_all
```
[the remark this answers is not in the document]
Comment (mrnoisytiger) [reply keyboards/lucid/kbd8x_hs/rules.mk:28]: Just for my understanding, what is the purpose of this section? I do see this listing pretty frequently and have not found any documentation regarding it. Would love to be pointed in the right direction here.

File: keyboards/lucid/phantom_solder/rules.mk (hunk 3 above)
Comment (mrnoisytiger) [reply keyboards/lucid/phantom_solder/rules.mk:28]: Same question as above here.

**Comment by mrnoisytiger** [comment]
@zvecr I was taking a look at the broken PR Lint and it appears that removing the 
```
LAYOUTS = [...]
```
section is causing the issue. It appears that the `info.json` file is required to match the layouts defined in the `rules.mk`. Can you please clarify why the `LAYOUTS =` in `rules.mk` needed to be removed?

File: keyboards/lucid/kbd8x_hs/rules.mk (hunk 4 above)
[the remark this answers is not in the document]
Comment (fauxpark) [reply keyboards/lucid/kbd8x_hs/rules.mk]: https://docs.qmk.fm/#/feature_layouts

**Comment by fauxpark** [comment]
@mrnoisytiger no, the thing that caused the lint to fail is the renaming of the `LAYOUT_all` macro in info.json to match the definition in <keyboard>.h. This allowed the lint check to match the two up and see that one has more elements than the other:
```
☒ lucid/phantom_solder: LAYOUT_all: Number of elements in info.json does not match! info.json:72 != LAYOUT_all:71
```

**Comment by mrnoisytiger** [comment]
Thanks @fauxpark. It appears I misread the linting result. I believe I have fixed the issue, as I had an errant key in Row 3. Apologies for the confusion there and thank you for clarifying the usage of the `LAYOUT = ` section.

**Comment by fauxpark** [comment]
Travis got rate limited, let me just poke it again.

**Comment by mrnoisytiger** [comment]
Thanks! Looks like checks passed. 😊

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**